python/triton/experimental/gluon/language/amd/_layouts.py

```diff
@@ -71,3 +69,17 @@ def stringify(x):
             return "_".join(map(str, x))
 
         return f"MFMA_{self.version}_{stringify(self.instr_shape)}_{self.transposed}_{stringify(self.warps_per_cta)}_{stringify(self.tiles_per_warp)}_{self.elem_type}_{stringify(self.ctas_per_cga)}_{stringify(self.cta_split_num)}_{stringify(self.cta_order)}_MFMA"
+
+    def verify(self):
+        assert self.version >= 0 and self.version <= 4, "version must be in the [0, 4] range"
```

Comment:
I don't think we accept version == 0. I see you copied this from the verifier. Can you fix there too?